azure-pipelines.yml

```diff
@@ -93,9 +93,9 @@ jobs:
 
     - template: etc/ci/azure-posix.yml
       parameters:
-          job_name: mac1014
```

Comment:
IMHO going from 10.14 to 11 is too big of a version jump at once..... what is the oldest version available after 10.14? use this one instead